packages/emf/backend/sirius-components-emf-forms/src/main/java/org/eclipse/sirius/components/emf/forms/LocalDateIfDescriptionProvider.java

```
@@ -0,0 +1,149 @@
+        };
+    }
+
+    private DateTimeDescription getDateTimeDescription() {
+        return DateTimeDescription.newDateTimeDescription(DATE_DESCRIPTION_ID)
+                .idProvider(new WidgetIdProvider())
+                .targetObjectIdProvider(this.semanticTargetIdProvider)
+                .labelProvider(this.getLabelProvider())
+                .stringValueProvider(this.getValueProvider())
+                .newValueHandler(this.getNewValueHandler())
+                .diagnosticsProvider(this.propertiesValidationProvider.getDiagnosticsProvider())
+                .kindProvider(this.propertiesValidationProvider.getKindProvider())
+                .messageProvider(this.propertiesValidationProvider.getMessageProvider())
+                .type(DateTimeType.DATE)
+                .build();
+    }
+
+    private Function<VariableManager, String> getLabelProvider() {
+        return new EStructuralFeatureLabelProvider(EMFFormDescriptionProvider.ESTRUCTURAL_FEATURE, this.composedAdapterFactory);
+    }
+
+    private Function<VariableManager, String> getValueProvider() {
+        return variableManager -> {
+            var optionalEObject = variableManager.get(VariableManager.SELF, EObject.class);
+            var optionalEAttribute = variableManager.get(EMFFormDescriptionProvider.ESTRUCTURAL_FEATURE, EAttribute.class);
+
+            if (optionalEObject.isPresent() && optionalEAttribute.isPresent()) {
+                EObject eObject = optionalEObject.get();
+                EAttribute eAttribute = optionalEAttribute.get();
+
+                Object value = eObject.eGet(eAttribute);
+                if (value instanceof LocalDate localDate) {
+                    try {
+                        Instant instant = localDate.atStartOfDay(ZoneId.of("UTC")).toInstant();
+                        return DateTimeFormatter.ISO_INSTANT.format(instant);
+                    } catch (DateTimeException | NullPointerException e) {
+                        // Ignore
```

- Why catch a null pointer exception?
- Why ignore the exception?

---

I removed the try catch to be coherent with all others existing providers.
The exception will be catched higher and an error message will be displayed to the user